examples/tf/debugging/simple_spread/feedforward/decentralised/run_maddpg_scale_trainers_fixed_nets.py

```diff
@@ -0,0 +1,118 @@
+# python3
```

Comment:
I guess the MADDPG code should not be here?

Reply:
I branched off the maddpg branch so that the common functions that were changed would be there

Reply:
This should be addressed if we first merge the other MADDPG PR :+1: